docs/Developers/Newcomers_Guide.rst

```diff
@@ -312,6 +320,18 @@ Step 4. Sending Your Changes
    Your current branch will have an asterisk (\*) next to it. Ensure that there
    is no asterisk next to the master branch.
 
+   Also make sure that your local clone is up-to-date with the original coala
+   repository before pushing. To do that, run
+
+   ::
+
+         $ git fetch upstream
+         $ git checkout master
+         $ git rebase upstream/master
+
+   where ``upstream`` is the original coala remote. You may need to force
+   push the repository the first time after rebasing.
```

Comment:
Actually first explaining what rebase actually does and then giving them how to rebase is better.

Reply:
That is already covered in http://api.coala.io/en/latest/Developers/Git_Basics.html

Reply:
coala recommends working in a branch different than master
so dont you think it is better to merge the master with upstream/master as it is just going to be a fast forward merge and rebase the 'branch-of-development' with the local master
This way one is only required to force push the development branch and simply push the master branch

Reply:
Yup, @Nshul is correct here.  We dont want newcomers committing to master, so these instructions need to be modified slightly.